### PR TITLE
added modal for google map

### PR DESCRIPTION
**Pull request by trba1810**
So i've created modal in location page on button with location icon,i left one working example and one not working without API_KEY from google. Do you want to generate api key or should i try to find different solution?

### CODE REVIEW

**Review by anicmarko9 — COMMENTED** [review]
You can generate API key, sure why not. I don't think that somebody is going to use it and abuse it in some kind of mean matter.

Thanks for your contribution, just watchout for responsiveness on smaller screens (320px), make it more human like. I'll wait for you to add API key and fix design, and I'll merge this into main.

**Comment by trba1810** [comment]
> You can generate API key, sure why not. I don't think that somebody is going to use it and abuse it in some kind of mean matter.
> 
> Thanks for your contribution, just watchout for responsiveness on smaller screens (320px), make it more human like. I'll wait for you to add API key and fix design, and I'll merge this into main.

Hey i've added custom height to tailwind css so we can use xs tag for future small screens problems ,but items still won't properly adjust to small screens even tho i tried couple of solutions. Can you help with that?? As for API-KEY even tho it's free someone need to leave credit card and that someone should be project owner. If you don't want that i can just use any map as placeholder.

File: tailwind.config.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ module.exports = {
       fontFamily: {
         heebo: ['Heebo', 'sans-serif'],
       },
+      screens: {
+        xs: { max: '639px' }, // max-width
+      },
```
Comment (anicmarko9) [inline tailwind.config.js:12]: We shouldn't worry about this. Just make sure you add bunch of classes to TSX files. Something similar to every other page.

**Review by anicmarko9 — APPROVED** [review]
I'll add google API key sometime when I get free time. Thanks for your contribution.

**Pull request merged**